magento/magento2
# Prevent page scroll jumping when product gallery loads

### Description (*)

During the investigation of the issue linked below I was able to pinpoint the exact moment of scroll jumping to fotorama's `that.resize` function which for some reason resets all of its major elements' width and height dimensions. I analysed the code and concluded that since current element height seems to be completely ignored in the whole script and is only set either based on settings or its width and ratio, the code that resets it can be removed. Please not that this can be also true for width but I left it here just to be safe as it doesn't contribute to page jumping.

In addition, `min-height` was added to prevent initial jump of the gallery when static image is being replaced by fotorama's HTML.

### Fixed Issues (if relevant)

1. magento/magento2#10518: Mobile product page image jumps
2. magento/magento2#21717: Product view page scrolls up randomly on mobile device

### Manual testing scenarios (*)

1. Open product page with images in the gallery in mobile view.
2. Scroll to the middle of the static image before gallery loads.
3. Page should no longer jump to the top once gallery initializes.

### Questions or comments
Please be sure to test this change on multiple browsers and devices including resizing the window through different resolutions as fotorama's configuration changes between breakpoins.

### Contribution checklist (*)
